index.js

```diff
@@ -287,8 +287,17 @@ function init (ssb, config) {
         tribeCreate(opts, (err, data) => {
           if (err) return cb(err)
 
+          const linkOpts = {
+            group: parentGroupId,
+            subGroup: data.groupId
+          }
+
+          if (opts && opts.admin) {
```

Comment:
hoping you might have a better way of doing this @mixmix

Reply:
alas no!